client/my-sites/plans/index.js

```diff
@@ -38,6 +38,14 @@ export default function() {
 		);
 
 		page(
+			'/plans/compare/:product/:domain',
+			retarget,
```

Comment:
@drewblaisdell looks like you had built this retargeting functionality prior to open-sourcing. Can you go back through and add some documentation to explain what it does, why, and when we need it? I see that it seems to push data into a queue that will eventually be used for purposes of tracking for facebook somehow. I'm not sure what the plans/upgrade routes have to do with facebook in particular. And the original PR references both facebook and google, but I'm not sure if that's accurate. Also not sure how the word `retarget` applies to the action we are taking.